modules/dashboard/widgets/dashboard-notepad.php

```diff
@@ -34,31 +34,29 @@ public function init() {
 	public function handle_notepad_update() {
 		global $pagenow;
 
-		if ( 'index.php' != $pagenow
-		|| ( empty( $_POST['action'] ) || 'dashboard-notepad' != $_POST['action'] ) )
+		if ( 'index.php' !== $pagenow || empty( $_POST['action'] ) || 'dashboard-notepad' !== $_POST['action'] ) {
 			return;
+		}
 
 		check_admin_referer( 'dashboard-notepad' );
 
-		if ( ! current_user_can( $this->edit_cap ) )
+		if ( ! current_user_can( $this->edit_cap ) ) {
 			wp_die( EditFlow()->dashboard->messages['invalid-permissions'] );
+		}
 
-		$current_id = (int) $_POST['notepad-id'];
-		$current_notepad = get_post( $current_id );
-		$new_note = array(
-				'post_content'           => wp_filter_nohtml_kses( $_POST['note'] ),
-				'post_type'              => self::notepad_post_type,
-				'post_status'            => 'draft',
-				'post_author'            => get_current_user_id(),
-			);
-		if ( $current_notepad
-			&& self::notepad_post_type == $current_notepad->post_type
-			&& ! isset ( $_POST['create-note'] ) ) {
-			$new_note['ID'] = $current_id;
-			wp_update_post( $new_note );
-		} else {
-			wp_insert_post( $new_note );
+		$note_data = array(
+			'post_content' => wp_filter_nohtml_kses( $_POST['note'] ),
```

Comment:
This is getting picked up by [PHPCS](https://travis-ci.org/Automattic/Edit-Flow/jobs/637800481?utm_medium=notification&utm_source=github_status) (scroll to the bottom of the output), can you address?